### ISSUE TITLE
Set up Travis.CI to automatically push on commit

### ISSUE THREAD

**Issue by marcoscaceres** [ResponsiveImagesCG/responsiveimages.org#75]
Manually having to push the site to dreamhost is no fun. It would be really nice if Travis.ci could just ftp all the changes up.

**Comment by marcoscaceres** [comment]
Anyone got Travis.ci experience?

**Comment by davidak** [comment]
should be easy: https://docs.travis-ci.com/user/deployment/custom/

**Comment by marcoscaceres** [comment]
@davidak, you feel like putting together a PR for us?

**Comment by davidak** [comment]
@marcoscaceres Yes. Could you commit the encrypted credentials (FTP_USER:FTP_PASSWORD) and give me the server domain and website path (ftp://sitename.com/directory/myfile)?

https://docs.travis-ci.com/user/encryption-keys/
